runtime/src/execute_cost_table.rs

```diff
@@ -44,24 +44,24 @@ impl ExecuteCostTable {
         self.table.len()
     }
 
-    /// default program cost, set to ComputeBudget::DEFAULT_UNITS
-    pub fn get_default_units(&self) -> u64 {
-        DEFAULT_UNITS as u64
+    /// default program cost, set to ComputeBudget::DEFAULT_COMPUTE_UNIT_LIMIT
```

Comment:
cross-ref: https://github.com/solana-labs/solana/pull/27229/files#r948692755 #27229

Reply:
i think this is typo. correct one would be `DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT`